keyboards/bastardkb/scylla/keymaps/xyverz/readme.md

```diff
@@ -0,0 +1,22 @@
+# The Bastard Keyboard
```

Comment:
I think this file can be deleted? As it's just a copy of the keyboard-level readme.

Reply:
yes